### PR TITLE
Create admin account by rake 50035

### PR DESCRIPTION
**Pull request by abdullahallmehedi**

### CODE REVIEW

**Comment by Rownaknira** [comment]
ok

**Comment by SSalekin** [comment]
ok

**Comment by noor-framgia** [comment]
ok

File: lib/tasks/admin.rake [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace :admin do
+  desc "Create admin account"
+  task create: :environment do
+    admin = User.create! name: Faker::Name.name, email: Faker::Internet.email,
```
Comment (dieunb) [inline lib/tasks/admin.rake]: if you define as it, how to know admin account to login? =))

**Pull request merged**